projects/bp-gallery/src/components/common/editor/Editor.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import JoditEditor, { Jodit } from 'jodit-react';
+import { addPlugins } from './helpers/jodit-plugins';
+
+export interface EditorProps {
+  value: string;
+  config?: Partial<Jodit['options']>;
+  onChange: (newValue: string) => void;
+  onBlur: (newValue: string) => void;
+}
+
+addPlugins();
+
+const Editor = ({ value, config, onChange, onBlur }: EditorProps) => {
```

Comment:
Maybe add a comment along the lines of this?
```
// The sole purpose of this wrapper is to add the paste plugin
```
If that's incorrect, please correct me :)

Reply:
Done